CHANGES.md

```diff
@@ -1,3 +1,77 @@
+## stable-2.6.0
+
+This release introduces experimental distributed tracing support, adds request
```

Comment:
I don't think there's any good reason to call this experimental. cc @grampelberg

Reply:
I think at least some of how tracing is configured is subject to change.  For example, I believe the `config.alpha.linkerd.io/trace-collector-service-account` annotation was named with `alpha` to indicate that it's going away soon.  @ihcsim or @grampelberg can you confirm?